p2p/addressbook/store.go

```diff
@@ -104,8 +118,22 @@ func (a *AddrBook) decodeFrom(path string) error {
 	}
 	defer r.Close()
 
+	data, err := ioutil.ReadAll(r)
+	if err != nil {
+		return errors.Wrap(err, "error reading file")
+	}
+
+	if len(data) < 4 {
```

Comment:
what is 4? plz give it a meaning

Reply:
before put data to file, we calc crc32. In golang lib it return uin32, this is 4 bytes.
So in file goes something like `123{some_json}`

So here we checking, is length enough? If so - split to `123` and `{some_json}`, verify that json str is not corrupted. 
123 as uint32 takes 4 bytes